gen/calc.py

```diff
@@ -375,7 +375,8 @@ def calculate_config_yaml(user_arguments):
 
 
 def calculate_mesos_isolation(enable_gpu_isolation):
-    isolators = 'cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,docker/runtime,docker/volume'
+    isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,' +
+                 'docker/runtime,docker/volume,volume/sandbox_path')
```

Comment:
Maybe worth converting this to an array of strings and then `join()` them with a comma?